stream-chat-android-client/src/test/java/io/getstream/chat/android/client/api2/MoshiChatApiTest.kt

```diff
@@ -2176,6 +2178,25 @@ internal class MoshiChatApiTest {
         assertEquals(location, result.getOrThrow())
     }
 
+    @ParameterizedTest
+    @MethodSource("io.getstream.chat.android.client.api2.MoshiChatApiTestArguments#getUnreadCounts")
```

Comment:
To avoid IDE lint about "not used method", would be good if you can add a kdocs comment over this test mentioning the method name, something like:
```
/** This method use [MoshiChatApiTestArguments.getUnreadCounts] as a source of arguments. */
```